datajoint/schema.py

```diff
@@ -1,15 +1,59 @@
+from operator import itemgetter
 import pymysql
 import logging
 
-from . import conn, DataJointError, NoDefinitionError
+from . import conn, DataJointError
+from datajoint.utils import to_camel_case
 from .heading import Heading
 from .base_relation import BaseRelation
-from .user_relations import Part
+from .user_relations import Part, Computed, Imported, Manual, Lookup
 import inspect
+
 logger = logging.getLogger(__name__)
 from warnings import warn
 
 
+class SchemaFactory:
+    """
+    Creates the appropriate python schema class objects from tables in a database.
```

Comment:
Would "Creates UserRelation classes for tables in the database" be clearer?

We don't need to say that these are in python.  Also, "schema class objects" is probably more confusing since these are not schema classes but classes for relations.